site/doc/1.11/journal.md

```diff
@@ -460,6 +492,28 @@ that no matter how many assertions you add, you can't be sure the
 account does not contain some unexpected commodity. (We'll add support
 for this kind of total balance assertion if there's demand.)
 
+
+The asserted balance checks only the amount(s) of each listed commodity's
+balance within the (possibly larger) account balance. We could call this a
+partial balance assertion. This is compatible with Ledger, and makes it
+possible to make assertions about accounts containing multiple commodities
+without needing to manually track every commodity the account contains.
+
+To instead assert a balance to the exclusion of all other commodities, use the
+exact assertion form `==EXPECTEDBALANCE`. This ensures that the account does
```

Comment:
There is some repetition here. You could delete lines 490 to 502 or something ?

Reply:
That's because the original version snuck back in through one of the merges...  It's fixed in the split branch.